packages/js-utils/src/object-helpers/lib/isEqual.ts

```diff
@@ -1,19 +1,80 @@
-import { toString } from './toString';
+function getExtendedType(arg: unknown) {
```

Comment:
i'm not sure about al the unknown types. maybe you could use generics to make this a little more type-safe